CONTRIBUTING.md

```diff
@@ -67,8 +67,9 @@ within a single repository.
 
 #### Testing
 
-Canvas Kit uses [Jest](https://jestjs.io/) and [Enzyme](https://airbnb.io/enzyme/) to unit test our
-React components. Each and every component requires at least 80% unit test coverage.
+Canvas Kit uses [Jest](https://jestjs.io/) and
+[React Testing Library](https://testing-library.com/docs/react-testing-library/intro) to unit test
+our React components. Each and every component requires at least 80% unit test coverage.
```

Comment:
maybe we remove the 80% as well and mention to go to `Testing.md` for more info?